README.md

```diff
@@ -248,7 +248,7 @@ trust_return_value_nullability = false
     # to rename the generated function
     rename = "something_else"
     # to override the default safety assertions: "none", "skip",
-    # "in-main-thread"
+    # "not-initialized", "in-main-thread"
```

Comment:
This makes no sense for the default of the whole crate though. Only for specific functions